source/brailleTables.py

```diff
@@ -86,18 +122,28 @@ def addTable(
 	:param output: True if this table can be used for output, False if not.
 	:param input: True if this table can be used for input, False if not.
 	:param source: An identifier describing the source of the table.
+	:param inputForLangs: A set of languages available in NVDA or C{None}.
+	:param translationForLangs: A set of languages available in NVDA or C{None}.
 	"""
 	if not output and not input:
 		raise ValueError("input and output cannot both be False")
 	table = BrailleTable(fileName, displayName, contracted, output, input, source)
 	_tables[fileName] = table
+	if inputForLangs is not None:
+		for lang in inputForLangs:
+			_inputTableForLangs[lang] = table.fileName
+	if translationForLangs is not None:
+		for lang in translationForLangs:
+			_translationTableForLangs[lang] = table.fileName
```

Comment:
Could you add an if check here to ensure that no other table was registered for that lang? Otherwise, the registration would be silently overridden without warning. Or is this intentional?

Reply:
This was intentional. We can change it if needed, but I don't see that this is necessary.